apps/studio/src/features/dashboard/components/ResourceTable/ResourceTableMenu.tsx

```diff
@@ -70,10 +74,13 @@ export const ResourceTableMenu = ({
               >
                 Edit page settings
               </MenuItem>
+
               {/* TODO(ISOM-1552): Add back duplicate page functionality when implemented */}
-              <MenuItem isDisabled icon={<BiDuplicate fontSize="1rem" />}>
-                Duplicate page
-              </MenuItem>
+              <Can do="update" on={{ parentId }} ability={ability}>
```

Comment:
issue: should this be `create` and not `update`?

Reply:
updated kind sir